variants/backend-base/config/initializers/version.rb

```diff
@@ -1,12 +1,6 @@
 Rails.application.config.version = begin
-  `git describe --always --tag 2> /dev/null`.chomp
+  # checks ENV["SHA"] and the "./REVISION" file
+  OkComputer::AppVersionCheck.new.version
 rescue StandardError
   "N/A"
 end
-
-Rails.application.config.version_time = begin
-  time = Time.zone.parse(`git log -1 --format="%ad" --date=iso 2> /dev/null`)
-  time || Time.zone.now
-rescue StandardError
-  Time.zone.now
-end
```

Comment:
@joshmcarthur I couldn't find a cross-environment way to do this so I removed it, reasoning that it's not as useful as the `version` itself. Hopefully this doesn't make you too sad. Let's chat if you think we should keep it.